### DISCUSSION TITLE
Add example to docs using `pseudopeople` python package

### DISCUSSION THREAD

**Discussion by RobinL** [moj-analytical-services/splink#1361] [Ideas]
From @aflaxman https://github.com/moj-analytical-services/splink/pull/1358#issuecomment-1604704761:

> I've been meaning to reach out to your team about a related topic, because I've just gotten to beta on a python package for generating simulated data for cases like this: https://pseudopeople.readthedocs.io/
> 
> Might this be an opportunity to brainstorm potential pseudopeople integration into splink docs?  Let me know if
> 
> You can see a notebook using it with splink here: https://colab.research.google.com/drive/1YVnK1G9vk_RPqH4kv52eZj_qCC8UtWMS?usp=sharing

**Comment by RobinL** [comment]
I'd definitely be up for adding an example to the Splink docs that uses a dataset generated by pseudopeople. Do you have a suggestion of a pre-created dataset that we could use? Or would you suggest that the example both generates and then links a dataset? If it turns out well we could also consider adding to these proposed inbuilt datasets

**Comment by aflaxman** [comment]
Great!

I think that a single year of simulated tax data could make a good, minimal example of deduplication --- it includes multiple rows for individuals who held multiple jobs in a year.  Linking simulated decennial and tax data could make a good, minimal example of `link_only`, and it even extends to include the somewhat authentic canonicalization task of adding SSNs from tax data to the decennial data for future linking and analysis.

Here is some code to provide a more concrete demonstration of the direction I'm thinking: https://colab.research.google.com/drive/1-vikcDAKXw2pgdvTtV4sw1XSVTgNUzBY?usp=sharing

**Comment by aflaxman** [comment]
This has now been done! https://moj-analytical-services.github.io/splink/demos/examples/duckdb_no_test/pseudopeople-acs.html
